### DISCUSSION TITLE
#27. VATZ biweekly meeting at 2023-02-01 08:00:00 -05:00:00

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#389] [General]
### 1. Overall
<img width="1215" alt="image" src="https://user-images.githubusercontent.com/6308023/214420827-0fd5da2c-ea09-4112-beab-740644f48a05.png">

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
25 | 4/19 | 21
26 | 17/34 | 50

---

## In Previous VATZ biweekly meeting

1. Stop command on CLI 
  (ongoing PR https://github.com/dsrvlabs/vatz/pull/387)
2. Monitoring in seperate ...
3. CLI documents
- Add Arguments in documents. 

# Poll
- https://github.com/dsrvlabs/vatz/issues/390#issuecomment-1411341601
- https://github.com/dsrvlabs/vatz/issues/388#issuecomment-1411322087VA

**Comment by xellos00** [comment]
# Stargaze Status
> 13 -> 16
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/6308023/214386919-69db109e-428d-443d-90c6-b2e832c34a6c.png">
Let's Keep it more. 

<img width="802" alt="image" src="https://user-images.githubusercontent.com/6308023/216053223-e3b7003d-421d-4280-b3cf-77fe187193c4.png">

**Comment by xellos00** [comment]
## `VATZ`

<img width="908" alt="image" src="https://user-images.githubusercontent.com/6308023/214387231-1c59c7ab-02c0-467c-9bff-c4cf4ae9a760.png">

<img width="913" alt="image" src="https://user-images.githubusercontent.com/6308023/216050848-84a9b425-e0f2-4698-aff7-419d38e3c509.png">

**Comment by xellos00** [comment]
# 동요

- https://github.com/dsrvlabs/vatz/issues/391
   - https://github.com/dsrvlabs/vatz/issues/388 
   - https://github.com/dsrvlabs/vatz/issues/390
- https://github.com/dsrvlabs/vatz/issues/392

**Comment by rootwarp** [comment]
# 준교
- https://github.com/dsrvlabs/vatz/issues/388

**Comment by skonhwang** [comment]
# 승곤
- no issue

**Comment by kim201212** [comment]
# 형기 

- no issue

**Comment by gnongs** [comment]
## 진홍
- no issue

**Comment by heejin-github** [comment]
## 희진
- no issue

**Comment by xellos00** [comment]
@hqueue 

- README. update 
- Create 
Feel Free to give us feedback.

We need not to cherish that much as much as possible. 
---

@xellos00 
README.md 한국어. No need

**Reply by xellos00** [reply]
<img width="441" alt="image" src="https://user-images.githubusercontent.com/6308023/216058807-457dffb6-e365-4133-84b0-a15e50658540.png">

**Reply by xellos00** [reply]
@gnongs 
1.0 을 찍자, 한번은 끈고 가야한다.

**Reply by xellos00** [reply]
Record Official Plugins on README.md

**Reply by xellos00** [reply]
for VATZ's icon. 
@Jihoo-Designer is going to create one for us. 
but its priority is really low for now.
